service.onmute/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.onmute" version="0.0.1" name="OnMute" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.simplejson" version="3.3.0"/>
```

Comment:
AFAIK that's not needed anymore since v17, as the internal python implementation now is good enough? (not sure if that's the cause)